### PR TITLE
Add Events FPP interface

### PR DESCRIPTION
**Pull request by menaman123**
…e set

| | |
|:---|:---|
|**Create Events Interface #2584**|  |
|**No**|  |
|**No_**|  |

---
## Change Description

Created an Event Interface out of the standard event ports using #2255 as an example. There were only 2 event ports compared to 3 stated in the issue description. 

## Rationale

Part of #2578.

## Testing/Review Recommendations

None

## Future Work

Must work on the other interfaces.

### CODE REVIEW

File: Ref/SignalGen/SignalGen.fpp [hunk 1]
```diff
@@ -92,6 +86,8 @@ module Ref {
     # ----------------------------------------------------------------------
 
     include "Events.fppi"
+    include "../Interfaces/EventsInterface.fppi"
```
Comment (thomas-bc) [inline Ref/SignalGen/SignalGen.fpp]: This is the events section where events are to be defined. Interfaces should likely be their own section, or another type of grouping.
@LeStarch do we have a convention for this? Should we?

Comment (thomas-bc) [reply Ref/SignalGen/SignalGen.fpp]: After we have an agreement, we probably want to do the same change on all components across `Ref/` and `RPI/`

Comment (LeStarch) [reply Ref/SignalGen/SignalGen.fpp]: I agree. We should create an interfaces section and place this there.

Also, `"../Interfaces/EventsInterface.fppi"` should read `"../../Fw/Interfaces/EventsInterface.fppi"`

**Comment by menaman123** [comment]
I added an interfaces section

**Review by LeStarch — APPROVED** [review]

**Comment by LeStarch** [comment]
I corrected the merge conflict with the other interface PR we merged.

**Review by thomas-bc — APPROVED** [review]

**Comment by menaman123** [comment]
Thank you so much for helping me out in my first adventure in open source development. Is there anything else I should do?

**Comment by LeStarch** [comment]
Nope! Welcome to the Open Source world, and thanks for completing this!

**Pull request merged**